src/lib/avatar/providers/ReplicateAvatar.ts

```
@@ -0,0 +1,251 @@
+    }
+
+    const startTime = Date.now();
+    const model = (options as { model?: string }).model ?? DEFAULT_MODEL;
+
+    const imageBuffer = await this.resolveBuffer(options.image);
+    const audioBuffer = await this.resolveBuffer(options.audio);
+
+    const imageDataUri = `data:image/${this.detectImageType(imageBuffer)};base64,${imageBuffer.toString("base64")}`;
+    const audioDataUri = `data:audio/${this.detectAudioType(audioBuffer)};base64,${audioBuffer.toString("base64")}`;
+
+    let prediction;
```

Biome (2.4.15)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)